sdk/security_keyvault/src/keys/operations/unwrap_key.rs

```diff
@@ -0,0 +1,88 @@
+use crate::prelude::*;
+use azure_core::{headers::Headers, CollectedResponse, Method};
+use serde_json::{Map, Value};
+
+operation! {
+    UnwrapKey,
+    client: KeyClient,
+    name: String,
+    unwrap_key_parameters: UnwrapKeyParameters,
+    ?version: String
+}
+
+impl UnwrapKeyBuilder {
+    pub fn into_future(mut self) -> UnwrapKey {
+        Box::pin(async move {
+            // POST {vaultBaseUrl}/keys/{key-name}/{key-version}/decrypt?api-version=7.2
+            let version = self.version.unwrap_or_default();
+            let mut uri = self.client.keyvault_client.vault_url.clone();
+            let path = format!("keys/{}/{}/unwrapkey", self.name, version);
+
+            uri.set_path(&path);
+
+            let mut request_body = Map::new();
+            request_body.insert(
+                "value".to_owned(),
+                Value::String(String::from_utf8(self.unwrap_key_parameters.ciphertext).unwrap()),
```

Comment:
A strong goal of the SDK is to actively avoid using `unwrap` in the SDK unless we have validated predicates that will ensure the SDK will not cause a panic.

I believe `azure_core::error::Error` has implementations to automatically convert `std::string::FromUtf8Error` (the error returned from `String::from_utf8`) to `azure_core::error::Error`.

Please replace this and the `unwrap` calls below with returning an error.

Reply:
I wanted to do so. Yet, currently these operations are generated by a macro. Also I wanted the code to be inline with all the other operations and they do it that way.

Reply:
The `operation!` macro builds `UnwrapKey` to be the following:

```rust
pub type UnwrapKey = futures::future::BoxFuture<'static, azure_core::Result<UnwrapKeyResponse>>
```

Replacing `.unwrap()` calls in your PR with `?` handles converting the error types automatically.

Reply:
Note, I just looked at the other operations in this crate.  The existing operations also use `unwrap`.  I'll be creating a PR to address that throughout the rest of this crate today.

Reply:
@reenigneEsrever92 I apologize for the confusion, as the existing code in this crate operated in the same fashion as your PR.

I've created https://github.com/Azure/azure-sdk-for-rust/pull/1312 to address this issue through the rest of this crate.  Hopefully the PR I linked demonstrates what is needed for this PR.